generic/generic_python-1.2.3.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="Short description of ProjectX"
+DESCRIPTION="Long ProjectX description.
+Spanning several paragraphs, maybe
+ - containing
+ - bullet points etc.
+You can also have really long lines that do not fit in 80 characters and use \
+a backslash to wrap the line."
+HOMEPAGE="https://homepage/of/projectx.org"
+COPYRIGHT="2014 Developer name
+	2016 Another developer's name"
+LICENSE="MIT
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/HaikuArchives/ProjectX/archive/projectx-$portVersion.tar.gz"
+CHECKSUM_SHA256="000000000000000000000000000000000"
+SOURCE_DIR="$portVersionedName"
+PATCHES="projectx-$portVersion.patchset"
+ADDITIONAL_FILES="projectx.rdef"
+
+ARCHITECTURES="any"
```

Comment:
should update to show new use of 'all'?

Reply:
In most cases for the python packages this would be "any", but since we're slowing down on python packages (can be installed via pip), this would probably become obsolete (but still handy to have around).